se.trixon.Mapollage.yml

```diff
@@ -0,0 +1,53 @@
+id: se.trixon.Mapollage
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: start-mapollage
+separate-locales: false
+
+finish-args:
+  - --device=dri
+  - --filesystem=host
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+
+build-options:
+    env:
+        - name=mapollage
+cleanup:
+  - '/mapollage/start-mapollage'    
+  - '/mapollage/se.trixon.Mapollage.desktop'    
+  - '/mapollage/se.trixon.Mapollage.metainfo.xml'    
+
+modules:
+  - name: mapollage
+    buildsystem: simple
+    build-commands:
+      - install -dm755 /app/bin
+      - install -dm755 /app/$name
+      - cp -r * /app/$name/
+      - chmod -R a-s,go+rX,go-w /app/$name
+      - unzip -o -j $name/core/locale/core_$name.jar org/netbeans/core/startup/frame*
+      - install -Dm644 frame512_$name.png /app/share/icons/hicolor/512x512/apps/$FLATPAK_ID.png
+      - install -Dm644 frame256_$name.png /app/share/icons/hicolor/256x256/apps/$FLATPAK_ID.png
```

Comment:
none of these files are found in the archive.

Reply:
Strange, they end up in by build and are provided by
`unzip -o -j $name/core/locale/core_$name.jar org/netbeans/core/startup/frame*`